ci-operator/jobs/openshift/elasticsearch-operator/openshift-elasticsearch-operator-tech-preview-postsubmits.yaml

```diff
@@ -0,0 +1,56 @@
+postsubmits:
+  openshift/elasticsearch-operator:
+  - agent: kubernetes
+    branches:
+    - ^tech-preview$
+    cluster: build02
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/is-promotion: "true"
+      ci-operator.openshift.io/prowgen-controlled: "true"
+    max_concurrency: 1
+    name: branch-ci-openshift-elasticsearch-operator-tech-preview-images
+    spec:
+      containers:
+      - args:
+        - --gcs-upload-secret=/secrets/gcs/service-account.json
+        - --image-import-pull-secret=/etc/pull-secret/.dockerconfigjson
+        - --image-mirror-push-secret=/etc/push-secret/.dockerconfigjson
+        - --promote
```

Comment:
What does `--promote` mean here? Promoting images to somewhere? If yes do we want this?

Reply:
all "jobs" are generated